src/mesh/voxel_cloud.rs

```diff
@@ -164,6 +164,18 @@ impl VoxelCloud {
         self.voxel_dimensions
     }
 
+    /// Checks if the voxel cloud contains any voxel / volume
+    #[allow(dead_code)]
+    pub fn contains_voxels(&self) -> bool {
+        self.voxel_map.iter().any(|v| *v)
+    }
+
+    /// Checks if the voxel cloud contains any voxel / volume
+    #[allow(dead_code)]
+    pub fn is_empty(&self) -> bool {
+        self.voxel_map.is_empty()
+    }
```

Comment:
From outside the scope there is no good way of checking these. 
In some cases it might be more efficient or safer to check these before performing any other operation. So far we don't seem to have such situation (or maybe I've overlooked it).